### PR TITLE
Feature/play timeline

### PR DESCRIPTION
**Pull request by Bluesmile82**
Play timeline component
play

I realized the play functionality cant be done inside the component as we need the playing year outside. 

Some corrections needed to be applied in order to display the first and last year as in the designs

### CODE REVIEW

File: src/components/play-timeline/play-timeline-styles.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    color: $midnight-blue;
+
+    .timelineMarker {
+      position: absolute;
+      display: inline-block;
+      height: 20px;
+      bottom: -6px;
+      width: 4px;
+      border-radius: 2px;
+      background-color: $theme-secondary;
```
Comment (weberjavi) [inline src/components/play-timeline/play-timeline-styles.scss]: Maybe some `transition` would be nice to smooth the movement of the marker when the years change :)

**Comment by Bluesmile82** [comment]
Updated to don't show the tick if is out of bounds and created the animation as it should work in Indonesia. Not sure we can add the transition as is the left property the one I'm changing

**Comment by Bluesmile82** [comment]
This is also ready again

**Comment by aabdaab** [comment]
Maybe instead of `left` we could use `translate3d`, that should work well with transition.

**Comment by simaob** [comment]
I believe this is ready to review.

**Pull request merged**